### PR TITLE
Updated Use Cases

### PR DESCRIPTION
**Pull request by DuqueKarl**
Added more TomoChain partners and their websites

### CODE REVIEW

File: docs/general/use-cases.md [hunk 1]
```diff
@@ -6,5 +6,14 @@ Partners willing to run applications on top of Tomochain:
 - Supply Chain: [https://devery.io/](https://devery.io/)
```
Comment (etienne-napoleone) [inline docs/general/use-cases.md]: Can you remove this one? Devery was an example but is actually not a partner of us

File: docs/general/use-cases.md [hunk 2]
```diff
@@ -6,5 +6,14 @@ Partners willing to run applications on top of Tomochain:
...
+- Robotics Ecosystem: [https://kambria.io/](https://kambria.io/)
+- Open Banking: [https://orcaalliance.eu/](https://orcaalliance.eu/)
+- Food Traceability: [https://www.te-food.com/](https://www.te-food.com/)
+- Pharmaceutical Tracking: [https://www.farmatrust.com/](https://www.farmatrust.com/)
+- IT Solutions: [https://savvycomsoftware.com/](https://savvycomsoftware.com/)
+- Travel Network: [https://www.triip.me/](https://www.triip.me/)
+- Decentralized Domain Name: [https://www.portal.network/](https://www.portal.network/)
+
 
 To be updated ...
```
Comment (etienne-napoleone) [inline docs/general/use-cases.md]: Can you replace this by an invitation to consult https://tomochain.com/dapps/ ?

File: docs/general/use-cases.md (hunk 1 above)
Comment (DuqueKarl) [reply docs/general/use-cases.md]: Removed this application in the last commit

File: docs/general/use-cases.md (hunk 2 above)
Comment (DuqueKarl) [reply docs/general/use-cases.md]: Replaced "to be updated" with an invitation to consult dApps on the link https://tomochain.com/dapps/

**Pull request merged**